rawfile/bd2fs.py

```diff
@@ -204,7 +229,7 @@ def CreateVolume(self, request, context):
             )
 
         access_type = volume_capability.WhichOneof("access_type")
-        assert access_type == "mount"
+        assert access_type == "mount" or access_type == "block"
```

Comment:
We can use

```python
access_type in {"block", "mount"}
```

Reply:
Ah yes, forgot you can do that in python :)
Done